### ISSUE TITLE
unable to change admin password

### ISSUE THREAD

**Issue by bibby-nh** [Kraken-CI/kraken#112]
Using the latest release (v0.557), I was unable to change the default `admin` password.

The log viewer produced these messages.

```
2021-06-07 15:02:43 c72638098a84  server  api ERRO  Exception on /api/users/1/password [PUT]
2021-06-07 15:02:43 c72638098a84  server  api ERRO  Traceback (most recent call last):
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
2021-06-07 15:02:43 c72638098a84  server  api ERRO      response = self.full_dispatch_request()
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
2021-06-07 15:02:43 c72638098a84  server  api ERRO      rv = self.handle_user_exception(e)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
2021-06-07 15:02:43 c72638098a84  server  api ERRO      reraise(exc_type, exc_value, tb)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
2021-06-07 15:02:43 c72638098a84  server  api ERRO      raise value
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
2021-06-07 15:02:43 c72638098a84  server  api ERRO      rv = self.dispatch_request()
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
2021-06-07 15:02:43 c72638098a84  server  api ERRO      return self.view_functions[rule.endpoint](**req.view_args)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/connexion/decorators/decorator.py", line 48, in wrapper
2021-06-07 15:02:43 c72638098a84  server  api ERRO      response = function(request)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/connexion/decorators/security.py", line 327, in wrapper
2021-06-07 15:02:43 c72638098a84  server  api ERRO      return function(request)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/connexion/decorators/uri_parsing.py", line 144, in wrapper
2021-06-07 15:02:43 c72638098a84  server  api ERRO      response = function(request)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/connexion/decorators/validation.py", line 184, in wrapper
2021-06-07 15:02:43 c72638098a84  server  api ERRO      response = function(request)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/connexion/decorators/validation.py", line 384, in wrapper
2021-06-07 15:02:43 c72638098a84  server  api ERRO      return function(request)
2021-06-07 15:02:43 c72638098a84  server  api ERRO    File "/venv/lib/python3.8/site-packages/connexion/decorators/parameter.py", line 121, in wrapper
2021-06-07 15:02:43 c72638098a84  server  api ERRO      return function(**kwargs)
2021-06-07 15:02:43 c72638098a84  server  api ERRO  TypeError: change_password() missing 1 required positional argument: 'user_password'
```

Cheers.

**Comment by godfryd** [comment]
The problem has been fixed. The fix is present in the 0.566 version.

**Comment by bibby-nh** [comment]
hey thanks!
